### ISSUE TITLE
Call for Maintainers

### ISSUE THREAD

**Issue by tcannonfodder** [ruby-passkeys/.github#1]
# Help needed
- **Maintainers desperately needed, particularly:**
	- Security researchers
	- People with experience writing devise gems
- Documentation for [warden-webauthn](https://github.com/ruby-passkeys/warden-webauthn)
- Documentation \+ tests for [devise-passkeys](https://github.com/ruby-passkeys/devise-passkeys)
- Registration & reauthentication helpers inside [devise-passkeys](https://github.com/ruby-passkeys/devise-passkeys)
- Security audit for gems
- Security-focused discussions on:
	- Account-recovery techniques
	- Emergency passkey registrations
	- Reauthentication logic

---

If you want to help make all our apps and our users more secure; comment below so we can figure out how to best get you onboarded! 💪

**Comment by Vagab** [comment]
For sure

**Comment by tcannonfodder** [comment]
Yay! Is there a particular thing you’d like to work on? Otherwise, I know we need documentation for devise-passkeys, see: https://github.com/ruby-passkeys/devise-passkeys/issues/2

**Comment by Vagab** [comment]
@tcannonfodder thanks for the quick response! I'd like to get more familiar with the gem and the process as a whole, documentation sounds like a nice place to start. Maybe after that I'll move on to registration & reauthentication helpers

**Comment by tcannonfodder** [comment]
Sounds like a plan! Definitely reach out if you have any questions/need anything clarified! 😄

**Comment by shaneshort** [comment]
is this project still kicking along? I'd like to see if I can help out?

**Comment by tcannonfodder** [comment]
> is this project still kicking along? I'd like to see if I can help out?

Yep, it’s still kicking along & your help would be immensely appreciated!

Is there a particular part of the project you’d like to tackle? Otherwise, feel free to grab one of the open documentation issues. 💪💪

thanks for reaching out!
